developer-docs-site/docs/concepts/coin-and-token/aptos-token.md

```diff
@@ -183,3 +162,55 @@ The token related data are stored at both creator’s account and owner’s acco
 | `TokenStore` | The main struct for storing the token owned by this address. It maps `TokenId` to the actual token. |
 | `Token` | amount is the number of tokens. |
 | `TokenId` | `TokenDataId` points to the metadata of this token. The `property_version` represents a token with mutated `PropertyMap` from `default_properties` in the `TokenData`. |
+
+## Token lifecycle
+
+### Token Creation
+
+Every Aptos Token belongs to a collection. The developer first needs to create a collection through `create_collection_script` and then create the token belonging to the collection `create_token_script`.
+To achieve parallel `TokenData` and `Token` creation, a developer can create unlimited collection and TokenData where the `maximum` of the collection and `TokenData` are set as 0. With this setting, the token contract won’t track the supply of types of token (`TokenData` count) and supply of token within each token type. As the result, the `TokenData` and Token can be created in parallel.
```

Comment:
nit: unlimited collection => an unlimited collection

Separately, may be we should add aggregator support for token supply to the roadmap? This would enable parallel token minting while still keeping track of the supply.